File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/iperson-attachment.ts




export interface IPersonAttachment {
  
	PersonID: number;
	FileID: number;
	FileName: string;
	FileLocation: string;
	PersonAttachmentTypeID: number;
	PersonAttachmentType: string;
	Description: string;
	IsDeleted?: boolean;
	ModifiedByAppUserID: number;
	FullName: string;
	ModifiedDate: Date;
	FileContent: number[];
	FileHash: number[];
	FileSize: number;

}

File: GTTS/INL.PersonService.Models/IPersonAttachment.cs
using System;

namespace INL.PersonService.Models
{
    public interface IPersonAttachment
    {
        long PersonID { get; set; }
        int FileID { get; set; }
        string FileName { get; set; }
        string FileLocation { get; set; }
        int PersonAttachmentTypeID { get; set; }
        string PersonAttachmentType { get; set; }
        string Description { get; set; }
        bool? IsDeleted { get; set; }
        int ModifiedByAppUserID { get; set; }
        string FullName { get; set; }
        DateTime ModifiedDate { get; set; }
        byte[] FileContent { get; set; }
        byte[] FileHash { get; set; }
        int FileSize { get; set; }
    }
}
